app/routes/about-us/route.tsx

```diff
@@ -1,3 +1,22 @@
+import styles from './about-us.module.scss';
+
 export default function AboutUsPage() {
-    return <h1>About us</h1>;
+    return (
+        <div className={styles.root}>
+            <div className={styles.aboutSection}>
+                <h1 className={styles.header1}>We are ReClaim</h1>
+                <p className={styles.header2}>A women-owned local business</p>
+                <p className={styles.description}>
```

Comment:
Don't use `<p>` elements. They don't allow putting block elements inside of them, and it's generally not something that people using a visual editor can understand or handle correctly.